src/index.js

```diff
@@ -1,5 +1,6 @@
 import valueParser from 'postcss-value-parser';
-import { parser } from './parser';
+
+import { parser } from './parser'; // eslint-disable-line
```

Comment:
This is due to `parser` only being available in the `dist` folder, it's automatically generated.